lib/tutorials/en_US/server-methods.md

```diff
@@ -1,21 +1,40 @@
-## Server methods
+# Server Methods
 
 _This tutorial is compatible with hapi v17_
 
-Server methods are a useful way of sharing functions by attaching them to your server object rather than requiring a common module everywhere it is needed. To register a server method, you call [`server.method()`](https://hapijs.com/api#server.method()). There are two different ways to call this function. You can call it with the signature `server.method(name, method, [options])`, for example:
+- [Overview](#overview)
+- [server.method()](#server.method)
+    - [Name](#name)
+    - [Method](#method)
+- [Options](#options)
+    - [Cache](#cache)
+    - [Generate a Custom Key](#key)
+    - [Bind](#key)
+- [server.methods()](#server.methods)
```

Comment:
`server.methods` is an object.  Would suggest change this to something like `server.methods` or `server.methods.myMethodName()`.
```suggestion
- [server.methods](#server.methods)
```